lib/ui/recommendation.js

```diff
@@ -59,9 +65,10 @@ Recommendation.prototype = {
       this.events.publish('recommendation-created');
     });
 
-    this.events.subscribe('recommendation', this.show);
+    this.events.subscribe('recommendation', this.onData);
     this.events.subscribe('enter-key', this.navigate);
     this.events.subscribe('urlbar-change', this.hide);
+    this.events.subscribe('urlbar-change', this._pollController);
```

Comment:
Seems weird to be exposing an underscored "private-ish" method to external code. Better to remove the underscore